main/icicle-repl.hs

```diff
@@ -153,7 +155,7 @@ data Command
 
 defaultState :: ReplState
 defaultState
-  = (ReplState [] demographics (unsafeTimeOfYMD 1970 1 1) False False False False False False False False False False False False False False False False False False False False)
+  = (ReplState [] demographics (unsafeTimeOfYMD 1970 1 1) P.InlineByName False False False False False False False False False False False False False False False False False False False False)
```

Comment:
default for repl should probably stay as by-value, until we change over to the new syntax